src/shared/ui/Footer.tsx

```
@@ -32,9 +32,18 @@ const Footer = () => {
             <GitHubLogoIcon
               width={20}
               height={20}
-              className="text-slate-400 transition-colors duration-300 ease-in-out"
+              className="text-slate-400 transition-colors duration-300 ease-in-out hover:text-slate-500"
+            />
+          </a>
+          <a
+            href="https://www.linkedin.com/in/jaeppetto/"
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)